packages/ibm-products-web-components/examples/delete-and-remove/src/2-delete-connected-items.ts

```diff
@@ -0,0 +1,169 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2025, 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, LitElement } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { map } from 'lit/directives/map.js';
+
+import '@carbon/web-components/es/components/modal/index.js';
+import '@carbon/web-components/es/components/form/form-item.js';
+import '@carbon/web-components/es/components/text-input/text-input.js';
+import '@carbon/web-components/es/components/notification/toast-notification.js';
+import '@carbon/web-components/es/components/checkbox/checkbox.js';
+import '@carbon/web-components/es/components/link/link.js';
+
+import { getCurrentTime } from './utils';
+import styles from './delete-and-remove.scss?lit';
+import { launchSVG } from './assets/launchSVG';
```

Comment:
Just a note for the lit transformed svgs, you can import them directly from `@carbon/web-components`. For example:

```ts
import Close20 from '@carbon/web-components/es/icons/close/20';
```

Then we don't have to have an `assets` directory unless they're anything custom.